FirebaseTestProject/Models/Post.swift

```diff
@@ -13,21 +13,31 @@ struct Post: Identifiable, Equatable, FirebaseConvertable {
     let author: User
     let id: UUID
     let timestamp: Date
+    var imageURL: URL? {
+        get {
+            URL(string: imageURLString)
+        }
+        set {
+            imageURLString = newValue?.absoluteString ?? ""
```

Comment:
Should this set to an empty string if the `absoluteString` is `nil`?  Or would a `nil` value make more sense?

Reply:
I used an empty string because Firestore doesn’t work with `nil` values. This goes to the discussion in #8—the non-optional `imageURLString` property is a workaround to make our models compatible with Firestore.